packages/skip-nav/__tests__/skip-nav.test.tsx

```diff
@@ -11,23 +11,26 @@ describe("<SkipNavLink />", () => {
     });
   });
 
-  // TODO: Doesn't pass, not sure why
-  //   it("should focus the SkipNavContent on click", () => {
-  //     let { getByText } = render(<Layout />);
-  //     act(() => {
-  //       fireEvent.click(getByText("Skip Nav"));
-  //       fireEvent.keyDown(document, { key: "Tab" });
-  //       expect(getByText("Focus me")).toHaveFocus();
-  //     });
-  //   });
-  //   it("should work with a custom ID", () => {
-  //     let { getByText } = render(<Layout skipNavId="whatever" />);
-  //     act(() => {
-  //       fireEvent.click(getByText("Skip Nav"));
-  //       fireEvent.keyDown(document, { key: "Tab" });
-  //       expect(getByText("Focus me")).toHaveFocus();
-  //     });
-  //   });
+  it("should focus the SkipNavContent on click", () => {
+    let { getByText, getByTestId } = render(<Layout />);
+    const mainContent = getByTestId("main");
+    act(() => {
+      userEvent.tab();
+      fireEvent.click(getByText("Skip Nav"));
+      userEvent.tab({ focusTrap: mainContent });
+      expect(getByText("Focus me")).toHaveFocus();
+    });
+  });
+  it("should work with a custom ID", () => {
+    let { getByText, getByTestId } = render(<Layout skipNavId="whatever" />);
+    const mainContent = getByTestId("main");
+    act(() => {
+      userEvent.tab();
+      fireEvent.click(getByText("Skip Nav"));
+      userEvent.tab({ focusTrap: mainContent });
```

Comment:
I'm not sure I understand the use of the focus trap here, can you help me understand why it's being used in this context? From the docs I see

> components like `focus-trap-react` will not work with `userEvent.tab()` or jsdom. For that reason, the `focusTrap` option is available to let you ensure your user is restricted within a focus-trap.

Reading this makes me feel like the use of the focus trap is to "mock" instances where a focus trap would already be in use (e.g. Reach's [Dialog][1]), but that isn't the case with `skip-nav`.

[1]: https://github.com/reach/reach-ui/blob/c5234721d372e2ab2a7745475007605e78d18895/packages/dialog/src/index.tsx#L185

Reply:
Yeah I'm a little lost on this one as well. This seems like we're doing something we shouldn't need to do in order to get the test to pass.